aws-cache-helpers/src/processors/ddb-stream-cache-handler.ts

```diff
@@ -40,8 +36,7 @@ export const NewStreamCacheHandler = ({
             const setRsp = await momento.set(
               cacheName,
               recordKey,
-              JSON.stringify(r.dynamodb.NewImage),
-              defaultCacheTtl
```

Comment:
No TTL?

Reply:
I think when I originally wrote this we didn't have way to set default ttl on overall cache client used. Now we do and I changed it up so user can just pass in the client configured the way they want to use it including a default ttl to use. So now dont need to have it passed on each set request explicitly.